ios/ReactNativeCharts/RNChartViewBase.swift

```diff
@@ -277,19 +277,56 @@ open class RNChartViewBase: UIView, ChartViewDelegate {
                 if limitLineConfig["limit"].double != nil {
```

Comment:
Can you also add an android implementation for it? Keep android and iOS in consistent is important.

Reply:
ok